src/components/markdown/partial-styles/_body-text.scss

```diff
@@ -12,6 +12,9 @@ p,
 a,
 li {
     font-size: 0.875rem;
+    word-break: break-word;
```

Comment:
Doesn't seems to work when it's a Heading

https://github.com/Lundalogik/lime-elements/assets/50618208/c98d97ee-1323-492f-9dd5-9b0d4d49eeed

Reply:
🤗 great catch! I added a fixup!